README.md

```diff
@@ -1 +1,3 @@
-# -Inverted-Index-Api
\ No newline at end of file
+# -Inverted-Index-Api
```

Comment:
For the sake of proper documentation and easy navigation for a new user of your app.